tests/fireproof/repro-blocks.process.test.ts

```diff
@@ -0,0 +1,44 @@
+import { describe, it, expect } from "vitest";
+
+// Skip this entire suite when running inside a browser-like Vitest environment
+const isNode = typeof process !== "undefined" && !!process.versions?.node;
+const describeFn = isNode ? describe : describe.skip;
+
+// everything node-specific is imported lazily inside the test body
+
+/* eslint-disable no-console */
+
+async function runScriptOnce(iter: number) {
```

Comment:
you have to move code here don't call the script from here --- that makes no sense